presidio-analyzer/presidio_analyzer/recognizer_result.py

```diff
@@ -35,7 +35,7 @@ def append_analysis_explenation_text(self, text: str) -> None:
         if self.analysis_explanation:
             self.analysis_explanation.append_textual_explanation_line(text)
 
-    def to_json(self) -> str:
+    def to_json_string(self) -> str:
```

Comment:
Consider creating a `to_dict` method have the caller cast to string if needed.
```suggestion
    def to_dict(self) -> Dict:
```

Reply:
Done